python/delta_sharing/rest_client.py

```diff
@@ -244,15 +244,15 @@ def list_files_in_table(
         *,
         predicateHints: Optional[Sequence[str]] = None,
         limitHint: Optional[int] = None,
-        sampleFraction: Optional[float] = None,
+        sampleHint: Optional[float] = None,
     ) -> ListFilesInTableResponse:
         data: Dict = {}
         if predicateHints is not None:
             data["predicateHints"] = predicateHints
         if limitHint is not None:
             data["limitHint"] = limitHint
-        if sampleFraction is not None:
-            data["sampleFraction"] = {"precisionFrom": 0.0, "precisionTo": sampleFraction}
+        if sampleHint is not None:
```

Comment:
As I can observe, sampleHint is always starting at 0.0. Does it make sense to include the possibility of a lower bound as well?

Reply:
I was thinking that it is a more universal approach that the client does caching if needed. More specifically, the client can ask from 0 to X% and cache the files, and then, from 0 to Y%. Thanks to the cached results only the range from X% to Y% should be fetched. Besides with this approach we also cover the scenario that the data gets updated between calls.